### PR TITLE
awk: add print every nth line example

### PR DESCRIPTION
**Pull request by Sinan81**
Print every "n"th line starting from line "x"

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/awk.md [hunk 1]
```diff
@@ -21,3 +21,7 @@
 - Sum the values in the first column and pretty-print the values and then the total:
 
 `awk '{s+=$1; print $1} END {print "--------"; print s}' {{filename}}`
+
+- Print every third line starting from the first line
```
Comment (pxgamer) [inline pages/common/awk.md]: As mentioned by the bot, this just needs to end in a colon. Other than this, I can't see anything wrong :+1:

**Comment by Sinan81** [comment]
I used this so many times over the course of 10 years in preparing and processing of serial farming type calculations on supercomputers, also I have been using this for system administration purposes.

File: pages/common/awk.md (hunk 1 above)
Comment (Sinan81) [reply pages/common/awk.md]: Sure!

**Comment by Sinan81** [comment]
CLA signed!

**Comment by agnivade** [comment]
@Sinan81 - Interesting. I have never had to use this. But since others have not raised any objections, I will merge this :smile:

**Pull request merged**